backend/src/sockets/gameStart.js

```diff
@@ -0,0 +1,10 @@
+/**
```

Comment:
I added a start websocket in the fronted and backend for next round to work, it’s in my current PR

Reply:
oh okay

Reply:
My implementation is a little different, so still may need parts of your PR depending on how we want to implement things and what functionalities we need